docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md

```diff
@@ -0,0 +1,266 @@
+# Approov Token Python Plugin Quickstart
+
+This quickstart is for developers familiar with Tyk who are looking for a quick intro into how they can add [Approov](https://approov.io) into an existing project. Therefore this will guide you through the necessary steps for adding the Approov token check to an existing Tyk API Gateway as a Python plugin middleware.
+
+
+## TOC - Table of Contents
+
+* [Why?](#why)
+* [How it Works?](#how-it-works)
+* [Requirements](#requirements)
+* [Approov Setup](#approov-setup)
+* [Approov Token Check](#approov-token-check)
+* [Test your Approov Integration](#test-your-approov-integration)
+
+
+## Why?
+
+To lock down your API server to your mobile app. Please read the brief summary in the [README](/README.md#why) at the root of this repo or visit our [website](https://approov.io/product.html) for more details.
+
+[TOC](#toc---table-of-contents)
+
+
+## How it works?
+
+For more background, see the overview in the [README](/README.md#how-it-works) at the root of this repo.
+
+
+[TOC](#toc---table-of-contents)
+
+
+## Requirements
+
+To complete this quickstart you will need to already have an Tyk API Gateway created, and the Approov CLI tool installed.
+
+* [Tyk API Gateway](https://tyk.io/docs/getting-started/)
+* [Approov CLI](https://approov.io/docs/latest/approov-installation/#approov-tool) - Learn how to use it [here](https://approov.io/docs/latest/approov-cli-tool-reference/)
+
+[TOC](#toc---table-of-contents)
+
+
+## Approov Setup
+
+To use Approov with the Tyk API Gateway we need a small amount of configuration. First, Approov needs to know the API domain that will be protected. Second, the Tyk API Gateway needs the Approov Base64 encoded secret that will be used to verify the tokens generated by the Approov cloud service.
+
+### Configure API Domain
+
+Approov needs to know the domain name of the API for which it will issue tokens.
+
+Add it with:
+
+```bash
+approov api -add your.tyk-api-gateway.domain.com
+```
+
+Adding the API domain also configures the [dynamic certificate pinning](https://approov.io/docs/latest/approov-usage-documentation/#approov-dynamic-pinning) setup, out of the box.
+
+> **NOTE:** By default the pin is extracted from the public key of the leaf certificate served by the domain, as visible to the box issuing the Approov CLI command and the Approov servers.
+
+### Approov Secret
+
+Approov tokens are signed with a symmetric secret. To verify tokens, we need to grab the secret using the [Approov secret command](https://approov.io/docs/latest/approov-cli-tool-reference/#secret-command) and plug it into the Tyk API Gateway configuration to check the signatures of the [Approov Tokens](https://www.approov.io/docs/latest/approov-usage-documentation/#approov-tokens) that it processes.
+
+
+To retrieved the Approov secret you need to enable your `admin` role with:
+
+```bash
+eval `approov role admin`
```

Comment:
I thought you'd added a note in other quickstarts that a different syntax is needed on Windows?

Reply:
Only for the Azure quickstart. I will add it to all.